app/src/main/index.js

```diff
@@ -82,14 +82,14 @@ function sleep(ms) {
   return new Promise(resolve => setTimeout(resolve, ms))
 }
 
-function handleCrash(error) {
+function handleCrash(err) {
```

Comment:
going from a non abbreviated word to an abbreviation feels weird.

Reply:
`error` is a reserved word in some programming languages. Also it makes it easier to search retrieved error responses